apps/server/scripts/bundleCss.ts

```diff
@@ -0,0 +1,25 @@
+import { existsSync } from 'fs';
+import { writeFile } from 'node:fs/promises';
+import { defaultCss } from '../src/user/styles/bundledCss';
+import path from 'path';
+
+/**
+ * Script to write contents of bundledCss to override.css
+ */
+async function bundleCss() {
+  try {
+    const stylesDir = path.resolve(process.cwd(), 'src', 'user', 'styles');
+    const cssFile = path.resolve(stylesDir, 'override.css');
+
+    if (!existsSync(cssFile)) {
+      throw new Error('File does not exist');
+    }
+
+    await writeFile(cssFile, defaultCss, { encoding: 'utf8' });
+  } catch (error) {
+    console.error('Failed writing to CSS file: ', error);
+    process.exit(1);
```

Comment:
cleanup: I dont think we need this, the program will terminate after this line anyway and there is nothing catching this directive